tools/upload-perfdata.ps1

```diff
@@ -0,0 +1,39 @@
+<#
+
+.SYNOPSIS
+This appends the contents of a local file to an Azure storage blob.
+
+#>
+param (
+    [Parameter(Mandatory = $true)]
+    [string]$ContainerName,
+
+    [Parameter(Mandatory = $true)]
+    [string]$BlobName,
+
+    [Parameter(Mandatory = $true)]
+    [string]$FileName
+)
+
+Set-StrictMode -Version 'Latest'
+$ErrorActionPreference = 'Stop'
+
+$StorageAccountName = "xdpperfdata"
+
+Write-Verbose "Connecting to storage account $StorageAccountName"
+$StorageContext = New-AzStorageContext -StorageAccountName $StorageAccountName -SasToken $env:SAS_TOKEN
```

Comment:
Why are you uploading to Azure for this? I still think GitHub is the way to go.

Reply:
Yeah, however I don't want to permanently trash our repo's commit history until we commit to that.

Reply:
"commit to that", good one